### DISCUSSION TITLE
Can I turn off the internet for a Valetudo network?

### DISCUSSION THREAD

**Discussion by Bjohnson131** [Hypfer/Valetudo#1755] [Q&A / Support] [answered]
Hello, I have a soft-gapped network that I like to run all of my home automation on. This network has no access to anything outside my LAN, or very limited access to specific points. 

Will a Valuetudo robot work on a network with no internet connection?

**Comment by Hypfer** [comment, marked as answer]
Yes, as far as Valetudo itself is concerned, there are no internet requirements. That would kinda defeat the purpose.
The only thing to keep in mind is that if you want Valetudo-internal timers to work, you'll have to configure your own internal NTP server. If you want to schedule cleaning with your home automation software - which is highly recommended over the internal timers - that doesn't matter.

With some robot firmwares, the firmware will power-cycle the Wi-Fi module every 15 minutes if it's not able to ping the gateway on that network so make sure to not disable ICMP.
Dreames also like to ping `8.8.8.8` to check for internet connectivity but not being able to do so to my knowledge has no downsides apart from a different logline in a logfile that no one will ever read.

**Reply by Bjohnson131** [reply]
neat! Can the Valetudo API server be ran on my local network, as opposed to on my vacuum?

**Reply by Hypfer** [reply]
https://github.com/Hypfer/Valetudo/discussions/1732

**Reply by Bjohnson131** [reply]
thank you kindly :) You raise a good point, if it was able to be done, it would have been done that way to start with.

**Reply by bugwelle** [reply]
I've run my Dreame L10 Pro without internet connection for a week now. I only had to set the NTP server to my router (`fritz.box` in my case, as that router was given to me by my internet provider). So far, everything works fine.

Is there any way to figure out if a robot power-cycles the Wi-Fi module? e.g. blinking LED, etc.?

**Reply by M1cha** [reply]
@bugwelle A power cycle would cause a wifi disconnect which you'd be able to see in the logs of your fritzbox.
